cmd/server/grpc/main.go

```diff
@@ -351,3 +365,56 @@ func (s *server) Signal(stream pb.SFU_SignalServer) error {
 		}
 	}
 }
+
+// Relay endpoint implementation
+func (s *Server) Relay(ctx context.Context, request *pb.RelayRequest) (*pb.RelayResponse, error) {
+
+	conn, err := grpc.Dial(request.Sfu, grpc.WithInsecure(), grpc.WithBlock())
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "%s", err)
+	}
+	defer conn.Close()
+
+	sfuClient := pb.NewSFUClient(conn)
+
+	client, err := sfuClient.Signal(ctx)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "%s", err)
+	}
+
+	me := sfu.MediaEngine{}
+	me.RegisterDefaultCodecs()
+
+	t, err := sfu.NewWebRTCTransport(ctx, &sfu.Session{}, me, sfu.WebRTCTransportConfig{})
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "%s", err)
+	}
+
+	offer, err := t.CreateOffer()
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "%s", err)
+	}
+
+	if err = t.SetLocalDescription(offer); err != nil {
+		return nil, status.Errorf(codes.Internal, "%s", err)
+	}
+
+	log.Debugf("Send offer:\n %s", offer.SDP)
+	err = client.Send(&pb.SignalRequest{
+		Payload: &pb.SignalRequest_Join{
+			Join: &pb.JoinRequest{
+				Sid: request.Sid,
+				Offer: &pb.SessionDescription{
+					Type: offer.Type.String(),
+					Sdp:  []byte(offer.SDP),
+				},
+			},
+		},
+	})
```

Comment:
We will need to create a goroutine here to handle renegotiation + trickle ice. But lets address issues above first